Tests/SwiftFormatPrettyPrintTests/FunctionDeclTests.swift

```diff
@@ -700,4 +700,15 @@ public class FunctionDeclTests: PrettyPrintTestCase {
       """
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 21)
   }
+
+  func testDoesNotCollapseFunctionParameterAttributes() {
+    let input =
+    """
+    func foo<Content: View>(@ViewBuilder bar: () -> View) {
+      bar()
+    }
+    """
```

Comment:
Once you've made the change above, please add a test case with a parameter with two stacked property wrappers so we can verify that works as well.